PUBLISHING.md

```diff
@@ -4,87 +4,99 @@
+1. Notify QA that the alpha release is ready for testing.
+
+## Checklist for Production Release
 
-## Pre-release checklist
 - [ ] Build is working correctly, passes automated tests and self-QA
-- [ ] Release candidate is delivered for QA
+- [ ] Alpha was delivered for QA
 - [ ] Sprint column is free from QA reported blocking issues
 - [ ] QA approved the release
-- [ ] All hotfixes to previous releases have been merged into the release (and develop) branch
-- [ ] CHANGELOG.md is up to date
+- [ ] All hotfixes for issues discovered in alpha releases have been merged into the release (and develop) branch
+- [ ] CHANGELOG.md is up to date and approved by @holmesworcester
 - [ ] PM approved the release
 
+## Preparing a Production Release
 
-## Publishing instruction
-
-By the time release is ready, ask @holmes for <b>CHANGELOG.md</b> cosmetic review, then:
-1. Checkout to a branch named after the release version number
-2. Navigate to root project directory
-3. Update CHANGELOG.md file
-4. Use the following command (with proper release type !For alpha releases use `pre` prefix!):  
-  `npm run publish --release=[patch|minor|major|EXACT_VERSION]`
-5. Cherry-pick `Publish` and `Update packages CHANGELOG.md` commits into /develop
-6. Manually update release notes on the <a href='https://github.com/TryQuiet/quiet/releases' target='blank'>Releases Page</a>
-
-## Post-release checklist (alpha)
-- [ ] App is promoted and sent for review on a closed testing track in Google Play
+1. If not already done, generate a [Github Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#) and set it as `GH_TOKEN` environment variable (or include it in the command as shown below).
+1. Trigger a production release with `GH_TOKEN=<token> npm run publish --release=[patch|minor|major|<EXACT_VERSION>]` (e.g. to create a prerelease of a minor update `npm run publish --release=minor`) This should remove the `alpha.x` suffix, and create a release on the [Releases Page](https://github.com/TryQuiet/quiet/releases), and trigger Github Actions to deploy the release to the Google Play Store and App Store. If the semantic version is different from the alpha release, select no in the prompt to create new versions, and call the command again with the exact version.
+1. Manually update the release notes on the [Releases Page](https://github.com/TryQuiet/quiet/releases) with the changes included in the release. See [RELEASE_NOTES_GUIDE.md](RELEASE_NOTES_GUIDE.md) for guidance on writing release notes.
+1. Promote the release on the [Google Play Console](https://play.google.com/console/) to a production track.
+1. Promote the release on the [App Store Connect](https://appstoreconnect.apple.com/).
 
 ## Post-release checklist (production)
-- [ ] Download links are updated on a website and in README.md
+
+- [ ] Download links are updated on website and in README.md
```

Comment:
I think we can cut "and in README.md" because that download link now links to the downloads section on the website.